.github/ISSUE_TEMPLATE/proposal.yml

```diff
@@ -0,0 +1,40 @@
+name: Proposal
+description: Proposal for feature to include in Interop 2023
+labels: proposal
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to make a proposal for Interop 2023!
+
+      Please see the [Interop 2023 README](https://github.com/web-platform-tests/interop/blob/main/2023/README.md) for the timeline and what to expect from the process.
+- type: textarea
+  attributes:
+    label: Description
+    description: Brief description of the feature. Try to be clear about what parts of the feature and included and not.
+- type: textarea
+  attributes:
+    label: Specification
+    description: |
+      Specification for the feature. The feature should be defined by a standards-track specification from the following organizations:
+
+      - [IETF](https://www.ietf.org/) (Standard Track)
+      - [Khronos Group](https://www.khronos.org/)
+      - [TC39](https://tc39.es/) (Stage 2+)
+      - [W3C](https://www.w3.org/) (Recommendation Track)
+      - [WHATWG](https://whatwg.org/)
+
+      Other specifications will be considered on a case-by-case basis.
+- type: textarea
+  attributes:
+    label: Tests
+    description: Tests for the feature in web-platform-tests, on the [wpt.fyi](https://wpt.fyi/) dashboard. Other test suites will be considered on a case-by-case basis.
+- type: textarea
+  attributes:
+    label: Rationale
+    description: |
+      Reasons for including the feature. A few dimensions to consider are:
+
+      - Web developer sentiment from polls or surveys, for example, [MDN surveys](https://insights.developer.mozilla.org/) or [State of CSS](https://stateofcss.com/).
```

Comment:
Can we add something like "Examples of existing sites or libraries working around a missing feature or browser bug"?

Reply:
Yeah, I think that's a good idea, it's very direct evidence that a bug mattered.

Reply:
I've added "Examples of sites or libraries working around a missing feature or browser bug" now.